tests/drivers/build_all/sensor/app.overlay

```diff
@@ -146,71 +146,16 @@
  * pristine
  */
 
-/* disable device to conflict with i2c version */
```

Comment:
Can you explain why are these removed ?
Are they built elsewhere ?

Reply:
> Can you explain why are these removed ? Are they built elsewhere ?

I remove the disabling of i2c, since I want the test to be done forboth busses.
I do not recall why originally the test excluded I2C for the sensors declared on SPI as well, but I feel that was because multi-istance was not working properly (or not at all) at that time. If you see, I still left a couple of STM sensors (ism330dhcx and iis2dh) with I2C disabled, as there are still issues when both i2c and spi are instanced. I will solve those issue as soon as possible, but not in this PR

Reply:
:-). My bad, commented to quickly. I didn't see that nodes were disabled initially.

Reply:
> :-). My bad, commented to quickly. I didn't see that nodes were disabled initially.

Thanks. I just re-pushed with no changes, but after rebasing on latest main branch.
(your approval disappeared... :D )

Reply:
> I do not recall why originally the test excluded I2C for the sensors declared on SPI as well, but I feel that was because multi-instance was not working properly (or not at all) at that time. But now I want to test both cases.

Can you mention this in the commit message please?

Reply:
Thank you!